x/marker/simulation/genesis.go

```diff
@@ -17,15 +19,27 @@ import (
 // Simulation parameter constants
 const (
 	MaxTotalSupply         = "max_total_supply"
+	MaxSupply              = "max_supply"
 	EnableGovernance       = "enable_governance"
 	UnrestrictedDenomRegex = "unresticted_denom_regex"
 )
 
-// GenMaxTotalSupply randomized Maximum amount of supply to allow for markers
+// GenMaxTotalSupply is deprecated.
+// Deprecated: GenMaxTotalSupply is kept for backwards compatibility.
 func GenMaxTotalSupply(r *rand.Rand) uint64 {
```

Comment:
Since this is in the simulation ... do we even need to keep this at all? We should be able to strip out all the old stuff it feels like...

Reply:
Sure, I can remove all the simulation material related to MaxTotalSupply.